CorePayments/src/main/java/com/paypal/android/corepayments/TrackingEventsAPI.kt

```diff
@@ -46,6 +46,7 @@ internal class TrackingEventsAPI constructor(
             .put(KEY_MERCHANT_APP_VERSION, merchantAppVersion)
             .put(KEY_PLATFORM, "Android")
             .put(KEY_ORDER_ID, event.orderId)
+            .put(KEY_BUTTON_TYPE, event.buttonType)
```

Comment:
```suggestion
            .putOpt(KEY_BUTTON_TYPE, event.buttonType)
```
I'd go with [putOpt()](https://developer.android.com/reference/org/json/JSONObject#putOpt(java.lang.String,%20java.lang.Object)) here since the value can be `null`.

Reply:
done!

Reply:
@sshropshire as `orderId` is also nullable, I made the change there too.